### PR TITLE
fix: always pass timeout

### PR DESCRIPTION
**Pull request by n1ru4l**
### Background

pass timeout at the correct location (https://github.com/kamilkisiela/graphql-hive/commit/3100043bb1b22fc0ec988aba7eca5b96c6f17888)

### Checklist

- [ ] Input validation
- [ ] Output encoding
- [ ] Authentication management
- [ ] Session management
- [ ] Access control
- [ ] Cryptographic practices
- [ ] Error handling and logging
- [ ] Data protection
- [ ] Communication security
- [ ] System configuration
- [ ] Database security
- [ ] File management
- [ ] Memory management
- [ ] Testing

### CODE REVIEW

File: packages/services/cdn-worker/src/aws.ts [hunk 1]
```diff
@@ -128,6 +134,7 @@ export class AwsClient {
 
   async fetch(input: RequestInfo, init: AwsRequestInit): Promise<Response> {
     for (let i = 0; i <= this.retries; i++) {
+      console.log('GO');
```
Comment (saihaj) [inline packages/services/cdn-worker/src/aws.ts:137]: Remove this now?

**Review by saihaj — APPROVED** [review]

File: packages/services/cdn-worker/src/aws.ts (hunk 1 above)
Comment (n1ru4l) [reply packages/services/cdn-worker/src/aws.ts:137]: yup

Comment (n1ru4l) [reply packages/services/cdn-worker/src/aws.ts:137]:
```suggestion
      console.log('GO');
```

File: packages/services/cdn-worker/src/aws.ts [hunk 2]
```diff
@@ -138,6 +144,8 @@ export class AwsClient {
           return res;
         }
       } catch (error) {
+        console.log('no');
```
Comment (n1ru4l) [inline packages/services/cdn-worker/src/aws.ts:147]:
```suggestion
```

**Pull request merged**